platforms/corda-ent/configuration/roles/helm_component/templates/node.tpl

```diff
@@ -56,7 +56,15 @@ spec:
         baseDir: /opt/corda
       jarPath: bin
       configPath: etc
-      cordaJarMx: 1524
+      cordaJar:
+        memorySize: 1524
+        unit: M
+      volume:
+        baseDir: /opt/corda/base
```

Comment:
there are two volume defined in this tpl ( line 56)

Reply:
done chnages